### ISSUE TITLE
What is foundation model

### ISSUE THREAD

**Issue by tranh215980** [georg-wolflein/pathology-feature-extractors#4]
Dear @georg-wolflein ,

Thank you for organizing this great list together . I have some questions:

- In README you say interesting that: "Models trained on >100K slides may be considered foundation models and are marked in bold" why is that? what make CTransPath and REMEDIS not a FM? I see definition changes in many paper and struggle with meaning.

- Virchow 2 is describe as "DINOv2 (+ ECT and KDE)". What is ECT and KDE and how are these add in DINOv2?

**Comment by georg-wolflein** [comment]
Both great questions!

- You are right: there is no generally accepted definition in the literature regarding the scale at which a feature extractor can be called a "foundation model". I used an arbitrary threshold of 100K -- I think this is a sensible number, but there is no empirical backing. Essentially I just wanted a number to differentiate between models trained on (relatively) "small" datasets vs "large" datasets, and so these are easier to identify, I marked them in bold. 
- ECT = extended-context translation; KDE = kernel density estimator. Refer to Figure 2 in the [Virchow 2 paper](https://arxiv.org/pdf/2408.00738) for details. 

I hope that answers your questions.
